src/containers/shared/css/simpleTab.scss

```diff
@@ -36,10 +40,10 @@ $index-width: 308px;
 
       .label {
         color: $white;
-        font-size: 12px;
-        line-height: 20px;
-        text-transform: uppercase;
-        @include medium;
+        font-size: 14px;
```

Comment:
Move `font-size: 14px;` and `line-height: 20px;` to `.rows` since all the elements share the same styles.